### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by wsanchez**

### CODE REVIEW

File: .travis/mypy [hunk 1]
```diff
@@ -37,22 +37,23 @@ tmp="$(mktemp -t mypy.XXXX)";
...
+        -e '^src/klein/_plating.py:.* error: Function is missing a [a-z ]* annotation'                      \
+        -e '^src/klein/_plating.py:[0-9:]*: error: Need type annotation for '                               \
+        -e '^src/klein/_resource.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'               \
+        -e '^src/klein/test/_trial.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'             \
+        -e '^src/klein/test/py3_test_resource.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'  \
+        -e '^src/klein/test/test_app.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'           \
+        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'       \
+        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Need type annotation for '                      \
+        -e '^src/klein/test/test_resource.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'      \
+        -e '^src/klein/test/util.py:[0-9:]*: error: Function is missing a [a-z ]* annotation'               \
```
Comment (wsanchez) [inline .travis/mypy]: This error message comes in two flavors now ("…missing a type…") and ("…missing a return type…"); we still want to ignore both for now.

File: .travis/mypy [hunk 2]
```diff
@@ -37,22 +37,23 @@ tmp="$(mktemp -t mypy.XXXX)";
...
-        -e '^src/klein/test/_trial.py:[0-9:]*: error: Function is missing a type annotation'             \
-        -e '^src/klein/test/py3_test_resource.py:[0-9:]*: error: Function is missing a type annotation'  \
-        -e '^src/klein/test/test_app.py:[0-9:]*: error: Function is missing a type annotation'           \
-        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Function is missing a type annotation'       \
-        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Need type annotation for '                   \
-        -e '^src/klein/test/test_resource.py:[0-9:]*: error: Function is missing a type annotation'      \
-        -e '^src/klein/test/util.py:[0-9:]*: error: Function is missing a type annotation'               \
+        -e ': error: Argument "converter" to "attrib" has incompatible type'                                \
+        -e ': error: Method must have at least one argument'                                                \
+        -e ': error: The type alias to [A-Za-z]* is invalid in runtime context'                             \
```
Comment (wsanchez) [inline .travis/mypy]: This is just whitespace

File: .travis/mypy [hunk 3]
```diff
@@ -37,22 +37,23 @@ tmp="$(mktemp -t mypy.XXXX)";
...
+        -e ': note: Use "-> None" if function does not return a value'                                      \
```
Comment (wsanchez) [inline .travis/mypy]: New note from mypy, associated with other errors below.

File: src/klein/_session.py [hunk 4]
```diff
@@ -21,10 +21,10 @@
 )
 
 if TYPE_CHECKING:               # pragma: no cover
+    from mypy_extensions import Arg, KwArg, VarArg
     from twisted.web.iweb import IRequest
     from twisted.python.components import Componentized
-    from mypy_extensions import KwArg, VarArg, Arg
-    from typing import TypeVar, Awaitable, Dict, Text
+    from typing import Awaitable, Dict, Sequence, Text, TypeVar
```
Comment (wsanchez) [inline src/klein/_session.py]: The change here is importing Sequence, then re-ordering as if flake8-import-order checks this (it doesn't, though).

File: src/klein/_session.py [hunk 5]
```diff
@@ -21,10 +21,10 @@
...
     T = TypeVar('T')
```
Comment (wsanchez) [inline src/klein/_session.py]: This adds an import for Sequence.  Rest is re-ordering as if flake8-import-order checked this (though it doesn't).

File: src/klein/_session.py [hunk 6]
```diff
@@ -117,7 +116,7 @@ def procureSession(self, request, forceInsecure=False):
                 [it for it in [request.getCookie(cookie)
                                for cookie in [self._secureCookie,
                                               self._insecureCookie]] if it]
-            )
+            )  # type: Sequence[Text]
```
Comment (wsanchez) [inline src/klein/_session.py]: Updated mypy wants an annotation here.

File: src/klein/_session.py [hunk 7]
```diff
@@ -33,7 +33,6 @@
 
 
 
-
```
Comment (wsanchez) [inline src/klein/_session.py]: 3, not 4 newlines before a class def

**Comment by glyph** [comment]
My only concern here is that 740 doesn't work with the zope interface plugin that I really, really want to use :).  But I filed https://github.com/Shoobx/mypy-zope/issues/12 for that because progress is eternal.

**Pull request merged**